bin/load_db_scxa_marker_genes.sh

```diff
@@ -52,6 +52,14 @@ echo "Marker genes: Create data file for $EXP_ID..."
 rm -f $EXPERIMENT_MGENES_PATH/mgenesDataToLoad.csv
 for f in $(ls $EXPERIMENT_MGENES_PATH/$MGENES_PREFIX*$MGENES_SUFFIX); do
   k=$(echo $f | sed s+$EXPERIMENT_MGENES_PATH/$MGENES_PREFIX++ | sed s/$MGENES_SUFFIX// )
+  if [ -e $EXPERIMENT_MGENES_PATH/$EXP_ID.clusters.tsv ]; then
+    # check that k is present in the second column of $EXP_ID.clusters.tsv,
+    # if such a file exists.
+    if ! awk '{ print $2 }' $EXPERIMENT_MGENES_PATH/$EXP_ID.clusters.tsv | tail -n +2 | grep -q ^$k$; then
+      echo "Skipping k=$k as it is not available in $EXP_ID.clusters.tsv file."
+      continue
+    fi
+  fi
```

Comment:
Here, I understand the cluster number (k) is extracted from filenames and look for associated k exist EXP_ID.clusters.tsv. If it doesn't continue. So the missing clusters are accounted but not loaded in mgenesDataToLoad.csv.

Reply:
Yes, exactly.